robottelo/performance/perf_candlepin.py

```diff
@@ -0,0 +1,116 @@
+"""Test utilities for writing Candlepin tests
```

Comment:
I think that this module can be called just candleping then the namespace will be robottelo.performance.candlepin.